src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
```

Comment:
the websites says: "Class variables should never be declared public"
So, perhaps it should not public.